README.md

```diff
@@ -1,7 +1,7 @@
 # FEE YEAR-AP: A Social Media Analytics Platform
 
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](code_of_conduct.md)
-
+[![license](https://img.shields.io/badge/license-FEE-green)](https://fee.org/)
```

Comment:
This shouldn't be Apache 2.0? We have the license here:
https://github.com/Marketing-Automation-Systems/jellyfish/blob/master/License.txt